### PR TITLE
add new link

### PR DESCRIPTION
**Pull request by maxday**
Hi, I've just built a tool using your awesome archive :)

### CODE REVIEW

**Comment by maxday** [comment]
hi @igrigorik,
did you a chance to review this PR ? Let me know if something is not wrong 
Thanks

File: index.html [hunk 1]
```diff
@@ -164,6 +164,7 @@ <h2 id="resources"><a href="#resources">Research, visualizations, talks...</a></
           <br/>
 
           <ul>
+            <li>How to detect Github trending repo API, using GithubArchive, Heroku, MongoDB and Github API ? <a href="https://maxday.github.io/trending/#JavaScript">Demo</a> - <a href="https://medium.com/@max.day/how-to-detect-github-trending-repo-api-using-githubarchive-heroku-mongodb-and-github-api-b3489efd9f3e">Medium blog post</a>.</li>
```
Comment (igrigorik) [inline index.html]: Can you please update GithubArchive to "GH Archive" to reflect the new name?

Comment (maxday) [reply index.html]: Done ! I also replace this in the Medium article. Thanks !

**Comment by igrigorik** [comment]
👍

**Pull request merged**